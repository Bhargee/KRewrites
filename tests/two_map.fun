("map" (fun d -> d - 1) (("map" (fun y -> y * 3)))
