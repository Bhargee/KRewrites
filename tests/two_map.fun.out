<T>
    <k>
        "map" (fun (#symExp(1) -> ((fun (d -> (d - 1))) ((fun (y -> (y * 
          3))) #symExp(1)))))
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
